cmd/config.go

```diff
@@ -22,25 +22,31 @@ import (
 	"github.com/spf13/cobra"
 
 	"github.com/pilosa/pilosa/ctl"
+	"github.com/pilosa/pilosa/server"
 )
 
 var Conf *ctl.ConfigCommand
 
 func NewConfigCommand(stdin io.Reader, stdout, stderr io.Writer) *cobra.Command {
 	Conf = ctl.NewConfigCommand(os.Stdin, os.Stdout, os.Stderr)
+	Server := server.NewCommand(stdin, stdout, stderr)
```

Comment:
That what I understand it's supposed to do. However I'm not able to get the config passed in, instead `pilosa config` printed out the default config. When I printed out config passed in `BuildServerFlags()`, it's actually the default config. 
```
&{DataDir: Host:localhost:10101 Cluster:{ReplicaN:1 Type:static Hosts:[] InternalHosts:[] PollingInterval:1m0s InternalPort: GossipSeed:} Plugins:{Path:} AntiEntropy:{Interval:10m0s} MaxWritesPerRequest:5000 LogPath:}
```
Here what's I did, please let me know if any steps that I missed:

my current config as following, which I ran with command `pilosa server --config ~/Documents/node1.conf`

```
data-dir = "/tmp/pil5_0"
bind = "127.0.0.1:10101"

[cluster]
  poll-interval = "2m0s"
  replicas = 2
  partitions = 128
  hosts = [
    "127.0.0.1:10101",
    "127.0.0.1:15001",
  ]
type = "gossip"
  internal-port = 14000
  gossip-seed = "127.0.0.1:14000"
  internal-hosts = [
    "127.0.0.1:14000",
    "127.0.0.1:14001",
]

[anti-entropy]
  interval = "1m0s"

[metric]
  service = "statsd"
  host = "127.0.0.1:8125"
  poll-interval = "0m15s"

```

On another iterm tab, I ran `pilosa config` and it printed out: 

```
data-dir = "~/.pilosa"
host = ":10101"
log-path = ""
max-writes-per-request = 5000

[anti-entropy]
  interval = 600000000000

[cluster]
  gossip-seed = ""
  hosts = []
  internal-hosts = []
  internal-port = ""
  polling-interval = 60000000000
  replicas = 1
  type = "static"

[plugins]
  path = ""
```